### PR TITLE
templates/form.html: Add Github fork me ribbon

### PR DESCRIPTION
**Pull request by Chirag0111**
This PR adds a Github fork me ribbon in form.html.

Closes https://github.com/manu-chroma/username-availability-checker/issues/126

### CODE REVIEW

**Review by utkarsh2102 — CHANGES_REQUESTED** [review]
Hey,
- Changes you made aren't really working. Did you test them locally?
- If possible, let's work on the commit message and mend it as per the [guidelines](https://coala.io/commit).
- Also, indent the code properly. That really helps.
- Include the snippets of the local test in the comments before submitting it for the review again. We'd know if that works ;)
- I'm extending the deadline by 24 hours. Please submit the same at the earliest. 
- Ping me on Gitter, Zulip or Telegram, if you need any help.

**Comment by manu-chroma** [comment]
this is duplicated code from ``status.html``

We can avoid duplicating by refactoring github icon css into main css file.

**Review by utkarsh2102 — APPROVED** [review]
Except for changing the branch, the changes look good to me.
I think he's having a little trouble with rebasing, that won't be a problem, I guess.
Also, the line exceeds by the usual length, but that's okay, I guess?
The changes are working fine.

File: templates/form.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html>
+
```
Comment (li-boxuan) [inline templates/form.html]: Unnecessary change?

**Review by manu-chroma — CHANGES_REQUESTED** [review]
github ribbon css is present in both ``status.html`` and ``form.html``

Please extract it out to ``main.css``

**Comment by Chirag0111** [comment]
@utkarsh2102, I have done the work.
- Added proper indentation, as asked by @utkarsh2102 and @jayvdb.
- Extracted css to `main.css`.

Please check the same.

**Review by utkarsh2102 — CHANGES_REQUESTED** [review]
I can't seem to find the hover effect working when tested locally.
Over to @manu-chroma.
Indentation looks fine now.

**Review by manu-chroma — APPROVED** [review]

**Comment by utkarsh2102** [comment]
Oh, that's awesome.
I will ask him to do so and would approve the task once he submits the same.

**Comment by Chirag0111** [comment]
Approved.

**Comment by Chirag0111** [comment]
Done.

image

**Comment by manu-chroma** [comment]
:tada:

**Pull request merged**